website/content/en/docs/component-guides/katib.md

```diff
@@ -6,197 +6,263 @@ weight = 20
-   ```
-
-8. Make the `default` SA an IRSA using the created IAM role from the previous step.
-
-   ```bash
-   eksctl create iamserviceaccount \
-   --cluster=$CLUSTER_NAME \
-   --name=default \
-   --namespace=${PROFILE_NAME} \
-   --attach-role-arn=$(aws iam get-role --role-name $PROFILE_NAME-$CLUSTER_NAME-role --query "Role.Arn" --output text) \
-   --region=$CLUSTER_REGION \
-   --override-existing-serviceaccounts \
-   --approve
-   ```
-
-## Verification steps
-
-9. Create the following Katib experiment yaml:
-
-   ```bash
-   cat <<EOF > experiment.yaml
-
-   apiVersion: kubeflow.org/v1beta1
-   kind: Experiment
-   metadata:
-     namespace: ${PROFILE_NAME}
-     name: test
-   spec:
-     objective:
-       type: maximize
-       goal: 0.90
-       objectiveMetricName: Validation-accuracy
-       additionalMetricNames:
-         - Train-accuracy
-     algorithm:
-       algorithmName: random
-     parallelTrialCount: 3
-     maxTrialCount: 12
-     maxFailedTrialCount: 1
-     parameters:
-       - name: lr
-         parameterType: double
-         feasibleSpace:
-           min: "0.01"
-           max: "0.03"
-       - name: num-layers
-         parameterType: int
-         feasibleSpace:
-           min: "2"
-           max: "5"
-       - name: optimizer
-         parameterType: categorical
-         feasibleSpace:
-           list:
-             - sgd
-             - adam
-             - ftrl
-     trialTemplate:
-       primaryContainerName: training-container
-       trialParameters:
-         - name: learningRate
-           description: Learning rate for the training model
-           reference: lr
-         - name: numberLayers
-           description: Number of training model layers
-           reference: num-layers
-         - name: optimizer
-           description: Training model optimizer (sdg, adam or ftrl)
-           reference: optimizer
-       trialSpec:
-         apiVersion: batch/v1
-         kind: Job
-         spec:
-           template:
-             metadata:
-               annotations:
-                 sidecar.istio.io/inject: "false"
-             spec:
-               containers:
-                 - name: training-container
-                   image: public.ecr.aws/z1j2m4o4/kubeflow-katib-mxnet-mnist:latest
-                   command:
-                     - "python3"
-                     - "/opt/mxnet-mnist/list_s3_buckets.py"
-                     - "&&"
-                     - "python3"
-                     - "/opt/mxnet-mnist/mnist.py"
-                     - "--batch-size=64"
-                     - "--lr=\${trialParameters.learningRate}"
-                     - "--num-layers=\${trialParameters.numberLayers}"
-                     - "--optimizer=\${trialParameters.optimizer}"
-               restartPolicy: Never
-   EOF
-   ```
-
-10. Create the experiment.
+4. Close the edit window. This will apply the configuration.
+
+
+### Example: S3 Access from Katib experiment pods
```

Comment:
Shall we move this example before _`katib-config` config map configuration_ section?  and say its only required if the algorithm pod needs to access AWS services

Reply:
I don't want to split the configuration section. 

I made the following change to that configuration section

#### `katib-config` config map configuration

**This configuration is only required if your Katib [algorithm](https://www.kubeflow.org/docs/components/katib/experiment/#search-algorithms-in-detail) pod needs access to AWS services.**

The [`katib-config`](https://www.kubeflow.org/docs/components/katib/katib-config/) contains configurations involving metrics collection, tuning algorithms, and early stopping algorithms.

Reply:
Sg!